src/Core/Services/StateMigrationService.cs

```diff
@@ -162,6 +162,8 @@ private async Task MigrateFrom2To3Async()
 
             var kdfType = await GetValueAsync<int?>(Storage.LiteDb, V2Keys.Keys_Kdf);
             var kdfIterations = await GetValueAsync<int?>(Storage.LiteDb, V2Keys.Keys_KdfIterations);
+            var kdfMemory = await GetValueAsync<int?>(Storage.LiteDb, V2Keys.Keys_KdfMemory);
+            var kdfParallelism = await GetValueAsync<int?>(Storage.LiteDb, V2Keys.Keys_KdfParallelism);
```

Comment:
@fedemkr Is this even necessary? Wasn't this for migrating data when we moved to state storage?